Dockerfile

```
@@ -35,6 +35,10 @@ RUN ["chmod", "-R", "u=rwX,go=rX", "/rootfs"]
 
 FROM scratch
 
+LABEL org.opencontainers.image.documentation='https://icinga.com/docs/icinga-db/latest/doc/01-About/' \
+      org.opencontainers.image.source='https://github.com/Icinga/icingadb' \
```

According to the [docs](https://github.com/opencontainers/image-spec/blob/v1.1.0/annotations.md#pre-defined-annotation-keys), `.source` should contain the "URL to get source code for building the image". Thus, maybe change it to this repository.

```suggestion
      org.opencontainers.image.source='https://github.com/Icinga/docker-icingadb' \
```

---

This is a tricky one. In most cases you got the Dockerfile as part of the source code repo. I would argue that most of the source code contained in the repo is located in that repo. Also you do find releases etc in the [icingadb repo](https://github.com/Icinga/icingadb). Ideally we could have both. If we link the [docker-icingadb repo](https://github.com/Icinga/docker-icingadb) most of the automations will not really be helpful (due to missing releases, code license etc).

---

Valid point. How about setting `org.opencontainers.image.url` to this repo, while `org.opencontainers.image.source` points to the Icinga DB source code repository?

---

Makes sense. Will do that.